index.js

```diff
@@ -0,0 +1,5 @@
+import CustomAttributeRegistry from './registry.js';
+
+var customAttributes = new CustomAttributeRegistry(document);
+
+export { customAttributes, CustomAttributeRegistry }
```

Comment:
@matthewp what do you think of using:

```js
export default customAttributes
export { CustomAttributeRegistry }
```
?
Import interface then:

```js
import customAttributes, { CustomAttributeRegistry } from 'custom-attributes';
```

Personally, like the idea of `customAttributes` being the default there, with `CustomAttributeRegistry` as a kind of extra utility.